core-drawer-panel.html

```diff
@@ -75,344 +75,425 @@
        *
-       * @attribute rightDrawer
-       * @type boolean
-       * @default false
+       * Listening for this event is an alternative to observing changes in the `selected` attribute.
+       * This event is fired both when a panel is selected and deselected.
+       * The `isSelected` detail property contains the selection state.
+       *
+       * @event core-select
+       * @param {Object} detail
+       * @param {boolean} detail.isSelected true for selection and false for deselection
+       * @param {Object} detail.item the panel that the event refers to
        */
-      rightDrawer: false,
+
+      published: {
+
+        /**
+         * Width of the drawer panel.
+         *
+         * @attribute drawerWidth
+         * @type string
+         * @default '256px'
+         */
+        drawerWidth: String,
+
+        /**
+         * Max-width when the panel changes to narrow layout.
+         *
+         * @attribute responsiveWidth
+         * @type string
+         * @default '640px'
+         */
+        responsiveWidth: String,
+
+        /**
+         * The panel that is being selected. `drawer` for the drawer panel and
+         * `main` for the main panel.
+         *
+         * @attribute selected
+         * @type string
+         * @default null
+         */
+        selected: {
+          reflect: true,
+          type: String
+        },
+
+        /**
+         * The panel to be selected when `core-drawer-panel` changes to narrow
+         * layout.
+         *
+         * @attribute defaultSelected
+         * @type string
+         * @default 'main'
+         */
+        defaultSelected: String,
+
+        /**
+         * Returns true if the panel is in narrow layout.  This is useful if you
+         * need to show/hide elements based on the layout.
+         *
+         * @attribute narrow
+         * @type boolean
+         * @default false
+         */
+        narrow: {
+          reflect: true,
+          type: String
+        },
+
+        /**
+         * If true, position the drawer to the right.
+         *
+         * @attribute rightDrawer
+         * @type boolean
+         * @default false
+         */
+        rightDrawer: Boolean,
+
+        /**
+         * If true, swipe to open/close the drawer is disabled.
+         *
+         * @attribute disableSwipe
+         * @type boolean
+         * @default false
+         */
+        disableSwipe: Boolean,
+
+        /**
+         * If true, ignore `responsiveWidth` setting and force the narrow layout.
+         *
+         * @attribute forceNarrow
+         * @type boolean
+         * @default false
+         */
+        forceNarrow: Boolean,
+
+        /**
+         * If true, swipe from the edge is disable.
+         *
+         * @attribute disableEdgeSwipe
+         * @type boolean
+         * @default false
+         */
+        disableEdgeSwipe: Boolean
+      },
+
+      listeners: {
+        click: 'onClick'
```

Comment:
Several people commented on the `Action` suffix and said a prefix is easier to pick out when debugging. I ran it by @morethanreal, and she was on board. I changed this in the style guide too.